src/ducks/balance/GroupPanel/index.js

```diff
@@ -0,0 +1 @@
+export { default as GroupPanel } from 'ducks/balance/GroupPanel/GroupPanel'
```

Comment:
nit: Barrel files are bad:
- They are bad for webpack treeshaking 
- They are bad for webpack performance (because in a large codebase we need to import hundreds / thousands barrel files, and it hurts perf)

Reply:
I hadn't realised that I was doing it to simplify my developer experience, but it shouldn't come at the sacrifice of the user experience

Reply:
It comes at the cost of your DX too. When I talk about webpack performance, this is on your machine ;). Also, now your IDE should be smart enough to be able to import the file correctly without barrel file. Your DX should not be impacted by that change.